marklogic-data-hub-central/ui/src/components/steps/steps.tsx

```diff
@@ -233,16 +234,12 @@ const Steps: React.FC<Props> = (props) => {
         <Tabs activeKey={currentTab} defaultActiveKey={DEFAULT_TAB} size={"large"} onTabClick={handleTabChange} animated={false} tabBarGutter={10}>
 
           <TabPane tab={(
-            <Tooltip getPopupContainer={() => document.getElementById("stepSettings") || document.body}
-              className={"basicTooltip"} style={ {wordBreak: "break-all"} }
-              title={(!isValid && currentTab !== "1") ? ErrorTooltips.disabledTab : null} placement={"bottom"}>Basic</Tooltip>
+            <HCTooltip text={(!isValid && currentTab !== "1") ? ErrorTooltips.disabledTab : ""} id="basic-tooltip" placement="bottom"><span>Basic</span></HCTooltip>
           )} key="1" disabled={!isValid && currentTab !== "1"}>
             {getCreateEditStep(props.activityType)}
           </TabPane>
           <TabPane tab={(
-            <Tooltip getPopupContainer={() => document.getElementById("stepSettings") || document.body}
-              className={"advTooltip"} style={ {wordBreak: "break-all"} }
-              title={(!isValid && currentTab !== "2") ? ErrorTooltips.disabledTab : null} placement={"bottom"}>Advanced</Tooltip>
+            <HCTooltip text={ErrorTooltips.disabledTab } id="basic-tooltip" placement="bottom"><span>Advanced</span></HCTooltip>
```

Comment:
@federicoRufrancosMarklogic  I think there's a missing condition for the text being rendered: `(!isValid && currentTab !== "2") `.